src/Entity/Service.php

```diff
@@ -150,6 +150,14 @@ protected function getErrandServiceIndex(ErrandService $errand_service) {
   public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
     $fields = parent::baseFieldDefinitions($entity_type);
 
+    $fields['name_synonyms'] = BaseFieldDefinition::create('string')
+      ->setTranslatable(TRUE)
+      ->setRevisionable(FALSE)
+      ->setLabel(new TranslatableMarkup('Name synonyms'))
+      ->setDescription(new TranslatableMarkup('Name synonyms for the service title.'))
+      ->setDisplayConfigurable('form', TRUE)
```

Comment:
maybe use this as used in description?
      ->setDisplayOptions('form', [
        'type' => 'readonly_field_widget',
      ])